packages/cli/commands/project/upload.js

```diff
@@ -43,81 +42,34 @@ exports.handler = async options => {
 
   await ensureProjectExists(accountId, projectConfig.name, { forceCreate });
 
-  const startPolling = async (tempFile, buildId) => {
```

Comment:
I ported this polling logic into the projects lib file so the dev command can use it too.